optaweb-vehicle-routing-backend/src/main/java/org/optaweb/vehiclerouting/OptaWebVehicleRoutingApplication.java

```diff
@@ -16,15 +16,18 @@
 
 package org.optaweb.vehiclerouting;
 
-import org.springframework.boot.SpringApplication;
+import org.optaweb.vehiclerouting.plugin.planner.Constants;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
 import org.springframework.scheduling.annotation.EnableAsync;
 
 @SpringBootApplication
 @EnableAsync
 public class OptaWebVehicleRoutingApplication {
 
     public static void main(String[] args) {
-        SpringApplication.run(OptaWebVehicleRoutingApplication.class, args);
+        new SpringApplicationBuilder(OptaWebVehicleRoutingApplication.class)
+                .properties("optaplanner.solver-config-xml=" + Constants.SOLVER_CONFIG)
```

Comment:
This is ok. Note that you can set it in application.properties too instead, fwiw.

Reply:
Ah, because the Solver config constant can be reused then. Yes, that makes sense.